hybrid/detailed_pv_plant.py

```diff
@@ -111,6 +111,7 @@ def system_capacity_kw(self, size_kw: float):
         :return:
         """
         self._system_model.value('system_capacity', size_kw)
+        self._financial_model.value('system_capacity', size_kw) # needed for custom financial models
```

Comment:
Will there be other properties that need this additional step?

Reply:
No, there are no other existing properties in DetailedPVPlant that need this extra step. However, there are the following values in the singleowner model that are updated after pvsamv1 is executed (and many more in singleowner and pvsamv1 that share the same name). These are currently updated in detailed_pv_plant::simulate_financials and power_source::simulate_financials for gen.
- batt_replacement_option
- en_standalone_batt
- om_batt_replacement_cost
- om_replacement_cost_escal
- gen

Reply:
Thanks for checking!